test/qan/profile/may-2015-01.json

```diff
@@ -1767,7 +2011,7 @@
         "Cnt": 112737,
         "Sum": 62.68778494838625,
         "Min": 0.000025,
-        "P5": null,
+        "P5": 0,
```

Comment:
Why this changed? Just verifying if it's ok.

Reply:
Just for simplification. It's rudimental and not used.